src/core/public/chrome/ui/header/header_logo.test.tsx

```diff
@@ -3,37 +3,41 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { EuiHeaderProps } from '@elastic/eui';
 import React from 'react';
 import { BehaviorSubject } from 'rxjs';
 import { mountWithIntl } from 'test_utils/enzyme_helpers';
 import { HeaderLogo, DEFAULT_DARK_LOGO, DEFAULT_LOGO } from './header_logo';
+import { BasePath } from '../../../http/base_path';
 
+const basePath = new BasePath('/base');
 const mockProps = () => ({
   href: '/',
+  basePath,
   navLinks$: new BehaviorSubject([]),
   forceNavigation$: new BehaviorSubject(false),
   navigateToApp: jest.fn(),
   branding: {},
+  theme: 'default' as EuiHeaderProps['theme'],
 });
 
-describe('Header logo ', () => {
-  describe('in default mode ', () => {
-    it('uses opensearch logo if no branding provided', () => {
+describe('Header logo', () => {
+  describe('when default-themed ', () => {
+    it('uses dashboards logo if no branding is provided', () => {
       const branding = {};
       const props = {
         ...mockProps(),
         branding,
       };
       const component = mountWithIntl(<HeaderLogo {...props} />);
       const img = component.find('.logoContainer img');
-      expect(img.prop('src')).toEqual(`/${DEFAULT_LOGO}`);
```

Comment:
Curiosity - prior to this change, did this actually load an asset from a different path than the next test case?

Reply:
The component depended on the `branding` to provide the `basePath` for the image. This test had an empty `branding` and as a result the test expected the URL to be `/${DEFAULT_LOGO}`.  The changed component uses the actual `basePath`.

Reply:
Aha! I knew something looked fishy.